conda/cli/main_env_remove.py

```diff
@@ -52,34 +54,28 @@ def configure_parser(sub_parsers: _SubParsersAction, **kwargs) -> ArgumentParser
     add_parser_solver(p)
     add_output_and_prompt_options(p)
 
-    p.set_defaults(func="conda.cli.main_env_remove.execute")
+    p.set_defaults(
+        func="conda.cli.main_remove.execute",
+        all=True,
+        channel=None,
+        features=None,
+        override_channels=None,
+        use_local=None,
+        use_cache=None,
+        offline=None,
+        package_names="",
```

Comment:
Before this `package_names=""` addition, the output for `conda env remove [environment]` was as follows:

```
Remove all packages in environment /conda/.../envs/remove_me:

Everything found within the environment /conda/.../envs/remove_me), including any conda environment configurations and any non-conda files, will be deleted. Do you wish to continue?
 (y/[n])? 
 ```
 
 After the addition, the command is much more like `conda remove [environment] --all`:
 
 ```
 Remove all packages in environment /conda/.../envs/remove_me:


## Package Plan ##

  environment location: 
/conda/.../envs/remove_me


The following packages will be REMOVED:

  bzip2-1.0.8-h80987f9_6
  ca-certificates-2024.7.2-hca03da5_0
  expat-2.6.2-h313beb8_0
  libcxx-14.0.6-h848a8c0_0
  libffi-3.4.4-hca03da5_1
  ncurses-6.4-h313beb8_0
  openssl-3.0.14-h80987f9_0
  pip-24.2-py312hca03da5_0
  python-3.12.4-h99e199e_1
  readline-8.2-h1a28f6b_0
  setuptools-72.1.0-py312hca03da5_0
  sqlite-3.45.3-h80987f9_0
  tk-8.6.14-h6ba3021_0
  tzdata-2024a-h04d1e81_0
  wheel-0.43.0-py312hca03da5_0
  xz-5.4.6-h80987f9_1
  zlib-1.2.13-h18a0788_1


Proceed ([y]/n)? y

Preparing transaction: done
Verifying transaction: done
Executing transaction: done
Everything found within the environment (/conda/.../envs/remove_me), including any conda environment configurations and any non-conda files, will be deleted. Do you wish to continue?
 (y/[n])?
 ```